src/test/java/edu/harvard/iq/dataverse/util/FileUtilTest.java

```diff
@@ -371,4 +371,33 @@ public void testHdf4File() throws IOException {
         assertEquals("application/octet-stream", contentType);
     }
 
+    @Test
+    public void testDetermineFileTypeROCrate() {
+        final String roCrateContentType = "application/ld+json; profile=\"http://www.w3.org/ns/json-ld#flattened http://www.w3.org/ns/json-ld#compacted https://w3id.org/ro/crate\"";
+        final DataFile rocrate = new DataFile(roCrateContentType);
+        
+        assertEquals(roCrateContentType, rocrate.getContentType());
+        assertEquals("RO-Crate metadata", FileUtil.getUserFriendlyFileType(rocrate));
+        assertEquals("Metadata", FileUtil.getIndexableFacetFileType(rocrate));
+
+        final File roCrateFile = new File("src/test/resources/fileutil/ro-crate-metadata.json");
+        if (roCrateFile.exists()) {
```

Comment:
Do we need to check for existence and have a sample file for this test? It should return just from seeing the name.

Reply:
I removed the file and adjusted the test, it works. Thanks!